src/PokemonGymOwner.java

```diff
@@ -0,0 +1,23 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer{
+    String name;
+    String town;
+    List<Pokemon> pokemons;
```

Comment:
Je overerft hier van PokemonTrainer, dus de "name" en "pokemons" variabelen zijn in deze klasse niet nodig. Die heb je ook meegenomen vanuit PokemonTrainer.